src/ts/components/timelinemarkershandler.ts

```diff
@@ -100,7 +99,7 @@ export class TimelineMarkersHandler {
 
       if (shouldRemoveMarker(markerPosition, markerDuration)) {
         this.removeMarker(marker);
-      } else {
+      } else if (markerPosition < 100) {
```

Comment:
```suggestion
      } else if (markerPosition <= 100) {
```

What if a marker is configured at `player.duration` (in a VOD of course)